### ISSUE TITLE
Poor rendering results when loading pre-trained 3DGS from .npy files

### ISSUE THREAD

**Issue by TSlus** [unique1i/SceneSplat#30]
Thanks for releasing this great dataset!

I'm trying to load the pre-trained 3DGS parameters from the provided .npy files to render RGB images using the standard rasterizer, but the rendering results look quite poor and unnatural.

I noticed the activation function mismatch for the physical values, so I have already applied the inverse activation for scale (using torch.log) to prevent the Gaussians from becoming huge and elongated. Here is my loading logic:

```python

def load_from_scene_splat(self, path):
    # breakpoint()
    xyz_path = os.path.join(path, "coord.npy")
    xyz = np.load(xyz_path)
    
    color_path = os.path.join(path, "color.npy")
    color = np.load(color_path)  # 你的发现: shape (N, 3), dtype=uint8
    
    opacity_path = os.path.join(path, "opacity.npy")
    opacities = np.load(opacity_path)
    
    scale_path = os.path.join(path, "scale.npy")
    scales = np.load(scale_path)
    
    quat_path = os.path.join(path, "quat.npy")
    rots = np.load(quat_path)

    N = xyz.shape[0]

    color_float = color.astype(np.float32) / 255.0

    SH_C0 = 0.28209479177387814

    features_dc = (color_float - 0.5) / SH_C0 

    features_dc = features_dc.reshape(N, 3, 1)

    features_extra = np.zeros((N, 3, 15), dtype=np.float32)

    if opacities.ndim == 1:
        opacities = opacities[:, np.newaxis]

    scales = np.log(scales + 1e-10)

    opacities = np.clip(opacities, 1e-5, 1.0 - 1e-5)
    opacities = np.log(opacities / (1.0 - opacities))

    # ==========================================
    self._xyz = nn.Parameter(torch.tensor(xyz, dtype=torch.float, device="cuda").requires_grad_(True))
    self._features_dc = nn.Parameter(torch.tensor(features_dc, dtype=torch.float, device="cuda").transpose(1, 2).contiguous().requires_grad_(True))
    self._features_rest = nn.Parameter(torch.tensor(features_extra, dtype=torch.float, device="cuda").transpose(1, 2).contiguous().requires_grad_(True))
    self._opacity = nn.Parameter(torch.tensor(opacities, dtype=torch.float, device="cuda").requires_grad_(True))
    self._scaling = nn.Parameter(torch.tensor(scales, dtype=torch.float, device="cuda").requires_grad_(True))
    self._rotation = nn.Parameter(torch.tensor(rots, dtype=torch.float, device="cuda").requires_grad_(True))

    self._opacity = self.inverse_opacity_activation(self._opacity)
    self._scaling = self.scaling_inverse_activation(self._scaling)

    self.active_sh_degree = 0
```

Despite this, the rendered output still looks incorrect. Could you advise if there is a specific/standard pipeline to correctly load your pre-trained .npy files for forward rendering? Are there any other hidden parameter transformations (e.g., quaternion normalization) that I might have missed?

Thanks in advance!

<img width="870" height="765" alt="Image" src="https://github.com/user-attachments/assets/44ce62ca-a192-42ed-a009-99f10a21b4be" />
<img width="709" height="864" alt="Image" src="https://github.com/user-attachments/assets/13480a2e-348d-48d7-b22e-e87c36873293" />

**Comment by ddghgf** [comment]
The dataset seems quite large, several terabytes. If the server can’t accommodate it, how is everyone handling this?
